### PR TITLE
Fix Examples layout page

### LINKED ISSUE #4888: Poor styling of examples' selectbox
Since the other issue is closed, here a copy of https://github.com/openlayers/ol3/issues/4883#issuecomment-187606395

The Development/Production-selectbox on the upper right corner is poorly styled. The text should be black and not white.

image

Furthermore, the impact of this box should be explained somewhere.

### PR DESCRIPTION
**Pull request by jonataswalker**
Let's give a more solid approach to the Examples page.

This PR fixes #4888 and remove the `position:absolute` made by #4831.

Now there's a `div` tying up the Logo `a#href` and the Search `form` (or Version `select`) together.

### CODE REVIEW

File: examples/resources/common.js [hunk 1]
```diff
@@ -19,10 +19,10 @@
   }
 
   if (window.location.host === 'localhost:3000') {
-    return;
+    // return;
```
Comment (ahocevar) [inline examples/resources/common.js]: Was this change meant to be in the pull request?

**Comment by ahocevar** [comment]
@jonataswalker If you can address my comment, I can include this fix in the 3.14.2 release, since we have a few more fixed regressions already.

File: examples/resources/common.js (hunk 1 above)
Comment (jonataswalker) [inline examples/resources/common.js]: Surely not.

**Comment by jonataswalker** [comment]
Addressed. Even after that [great explanation](https://github.com/openlayers/ol3/pull/4868#issuecomment-187785482) by Tim about rebasing and force pushing I'm still not sure:

I removed the unintended `// return;`, `git commit` and `git push --force origin branch-name`.

Is this all to do?

**Comment by bartvde** [comment]
You should rebase and then force push

```
git rebase -i HEAD~2
git push origin -f [branch_name]
```

**Comment by jonataswalker** [comment]
@bartvde thanks. Still in time for this one?

**Comment by ahocevar** [comment]
Still in time. There is one more unmerged regression fix.

**Comment by ahocevar** [comment]
But you still have two commits. When you do the rebase, change the word 'pick' in front of the 2nd commit to 'fixup'.

**Comment by ahocevar** [comment]
Alternatively, you can do something else in this case:
```
git reset HEAD^
git add .
git commit --amend
```
followed by a force push.

**Comment by jonataswalker** [comment]
I guess this is it, a single commit. :coffee:

**Comment by ahocevar** [comment]
Awesome, thanks @jonataswalker

**Pull request merged**